Yafc/Data/Sandbox.lua

```diff
@@ -35,6 +35,16 @@ function data:extend(t)
 	end
 end
 
+function data.extend(self, otherdata)
+	if self ~= data and otherdata == nil then
+		otherdata = self
+	end
+	for i=1,#otherdata do
+		local prototype = otherdata[i];
+		dataAdd(prototype.type, prototype.name, prototype);
+	end
+end
```

Comment:
```suggestion
require("__core__/lualib/dataloader.lua")
```
Thanks for figuring this out!

If we just load the same file Factorio uses, we'll get any new changes the devs make for free. (Maybe next they'll allow passing a single prototype instead of passing an array of prototypes?) On the other hand, the previous change to that file was in 1.1.10, so maybe they aren't planning to touch it again for a while.

With this `require`, it appears to me that lines 22-36 can also be deleted.